docs/getting_started/online_comp_getting_started.md

```diff
@@ -75,10 +75,14 @@ The output panel also displays results for the "Find in Folder" functionality. H
 
 ## Create your first program
 
-From the main toolbar, click the "New" button, and complete the program name field in the dialog box like on the image below.
+From the main toolbar, click the "New" button, select your target platform and choose a starter template from the drop-down list or select "Empty Program" to create a blank project. Complete the "Program Name" field like in the image below.
 
 <span class="images">![](images/create-program.png)<span>Creating a new program</span></span>
 
 Once you click "OK", the program is created in your workspace, and the mbed Online Compiler automatically opens it.
 
-The program is not empty. It contains two items - a main.cpp file and the mbed OS library. This is because the mbed OS library is applicable for every project based on an mbed hardware board. Additionally, the main.cpp file is the main file for every program. Thus, we decided to make the default new program template a small LED test that compiles quickly.
+If you chose to create your new program using a starter template, the program is not empty. It contains two items - a main.cpp file and the mbed OS library. This is because the mbed OS library is applicable for every project based on an mbed hardware board. Additionally, the main.cpp file is the main file for every program. 
```

Comment:
Please put "main.cpp" in code format with the ` symbol on either side of it. Please do this for the first and last sentence.